src/main/java/ua/samoplavskaya/HW_JSON/MongoTask.java

```diff
@@ -0,0 +1,44 @@
+package ua.samoplavskaya.HW_JSON;
+
+import com.mongodb.client.*;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+import com.mongodb.client.result.DeleteResult;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+
+public class MongoTask {
+    public static void main(String[] args) {
+        try (MongoClient mongoClient = MongoClients.create("mongodb://admin:admin123@localhost:27017")) {
+            MongoDatabase database = mongoClient.getDatabase("test");
+            MongoCollection<Document> collection = database.getCollection("orders");
+            Document doc = new Document("product", "Iphone 11")
+                    .append("price", 15000)
+                    .append("quntity", 3);
+            collection.insertOne(doc);
+
+            FindIterable<Document> documents = collection.find(Filters.gt("price", 30000));
+            try (MongoCursor<Document> cursor = documents.cursor()) {
+                while (cursor.hasNext()) {
+                    System.out.println(cursor.next());
+                }
```

Comment:
Minor note:

Since FindIterable alread is a Iterable object, we just can use for-loop with it, without getting a cursor (because it's a more low level abstraction)